src/Element.jl

```diff
@@ -24,3 +25,26 @@ n_elec_valence(attype::Element) = charge_ionic(attype)
 
 """Return the number of core electrons"""
 n_elec_core(attype::Element) = charge_nuclear(attype) - charge_ionic(attype)
+
+"""Radial local potential, in Fourier space: V(q) = int_R^3 V(x) e^-iqx dx"""
+function local_potential_fourier(e::Element, q::T) where {T <: Real}
+    if q == 0
+        return T(0) # assumes a compensating background
+    end
+    if e.psp === nothing
+        # All-electron => Use default Coulomb potential
+        # We use int_R^3 1/r e^-iqx = 4π / q^2
+        return -4T(π) * charge_nuclear(e) / q^2
+    else
+        # Use local part of pseudopotential defined in Element object
+        return eval_psp_local_fourier(e.psp, q)
+    end
+end
+function local_potential_real(e::Element, r::Real)
+    if e.psp === nothing
+        return 1/r
```

Comment:
Strongly against for this one. 1/r is more readable (in isolation) as 1 / r. I think spaces make sense for complex expressions but they should be used sparingly, eg for me a + b*x is better than a+b*x which is better than a + b * x

Reply:
Really?
```
return 1 / r
```
versus
```
return 1/r
```
I find the first better, but since I am used to having spaces around them I might be spoiled.

Reply:
Meh, I don't particularly care for such expressions as this one, so feel free to change it (and others I probably have missed) if you want and push to this branch. I'm more strongly attached to more complex expressions where the judicious usage (and non-usage) of spaces is important for readability.

Reply:
Ok. I'll do when implementing the Cohen-Bergstresser and changing the Element.